boards/arm64/imx8mp_evk/imx8mp_evk.dts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx8mp_a53.dtsi>
+
+/ {
+	model = "NXP i.MX8MP A53";
+	compatible = "fsl,imx8mp";
+
+	aliases {
+		uart-2 = &uart2;
+		uart-4 = &uart4;
+	};
```

Comment:
we've generally moved away from using aliases like this. do you still need these?

Reply:
Dropped.